test/unit/test_parameter_parser.py

```diff
@@ -1,35 +1,43 @@
-import re
 from textwrap import dedent
 from unittest import mock
 
 import pytest
 
-from exasol.ai.mcp.server.parameter_parser import parameter_list_pattern
-from exasol.ai.mcp.server.parameter_pattern import regex_flags
+from exasol.ai.mcp.server.parameter_parser import (
+    format_func_comment,
+    format_udf_comment,
+)
 
 
 @pytest.mark.parametrize(
-    "params_list",
-    ["( abc DOUBLE, xZZ varchar(200) )", '("my_real" REAL, "my_int" INTEGER)'],
-    ids=["non-quoted-names", "quoted-names"],
+    ["func_comment", "expected_formatted_comment"],
+    [
+        ("My function is awesome!", "My function is awesome!\nIn an SQL query"),
+        (None, "In an SQL query"),
+    ],
+    ids=["comment", "no-comment"],
 )
```

Comment:
Sorry, it looks messy in the changes because I moved some tests to the test_parameter_pattern.py. Something I forgot to do in the last PR.